### ISSUE TITLE
UART-flash process details unclear

### ISSUE THREAD

**Issue by myhobby2** [mikeeq/xiaomi_ax3200_openwrt#22]
In your instruction how to gain access via the uart-flash method (to-be found via 
https://github.com/mikeeq/xiaomi_ax3200_openwrt#uart-flash), it is not clear enough which image to flash in step 6 and where to find it. I've a recent RB01 without any telnet, but with your help gaining access to the UART was a piece of cake. That is working already.

**Comment by mikeeq** [comment]
You should flash the same stock firmware two times, I will update the readme.

**Comment by myhobby2** [comment]
I figured that out already ;-) but it is good to update this procedure. Successfully flashed 2 boxes via UART and TFTP (after a lot of hours). After a lot of struggle I used a Raspberry Pi as TFTP server on 192.168.31.100. That worked well but ran later in the process again in challenges because openWRT boots in a new network range (192.168.1.1), so my PI needed to be reconfigured to that range also... 

But your procedure works for a box manufactored in Nov 2021 with no telnet enabled ! Thanks
